data/management/commands/import_contributors_data.py

```diff
@@ -28,10 +29,11 @@ def import_data(self, contributor):
             )
             if created:
                 c.save()
-                print('\nContributor, {}, has been saved.'.format(c))
+                logging.debug('\nContributor, {}, has been saved.'.format(c))
         except Exception as ex:
-            print('\n\nSomething went wrong saving this contributor: {}\n{}'
-                  .format(login, str(ex)))
+            logging.debug(
```

Comment:
this is not a debug message.  this is important information we need to investigate.